pcweb/pages/docs/__init__.py

```diff
@@ -112,6 +113,21 @@ def exec_blocks(doc, href):
 outblocks = []
 
 
+manual_titles = {
+    "docs/library/datadisplay/callout-ll.md": "Callout Low",
+    "docs/database/overview.md": "Database Overview",
+    "docs/custom-components/overview.md": "Custom Components Overview",
+    "docs/api-routes/overview.md": "API Routes Overview",
+    "docs/client_storage/overview.md": "Client Storage Overview",
+    "docs/state/overview.md": "State Overview",
+    "docs/styling/overview.md": "Styling Overview",
```

Comment:
Similarly here - I think we have some logic on the prev/next buttons to modify any Overview page with the correct title. Maybe we can reuse the same function here